### PR TITLE
NEW(Output) Trim message output for large content

### PR DESCRIPTION
**Pull request by michalxo**
This should affect all JMS clients.
Not sure about other clients though.

### CODE REVIEW

File: jmslib/src/main/java/com/redhat/mqe/lib/JmsMessageFormatter.java [hunk 1]
```diff
@@ -148,6 +148,21 @@ public void addFormatJMS20(Message msg, Map<String, Object> result) throws JMSEx
...
+     * This is suitable for large messages
+     */
+    public Map<String, Object> formatMessageAsLargeInterop(Message msg, boolean hashContent) throws JMSException {
+        Map<String, Object> result = formatMessage(msg, hashContent);
+        addFormatInterop(msg, result);
+        result.put("id", removeIDprefix((String) result.get("id")));
+        result.put("user-id", removeIDprefix((String) result.get("user-id")));
+        result.put("correlation-id", removeIDprefix((String) result.get("correlation-id")));
+        result.put("group-sequence", changeMinusOneToZero((long) result.get("group-sequence")));
+        result.put("content", "<omitted due to large message size>");
```
Comment (michalxo) [inline jmslib/src/main/java/com/redhat/mqe/lib/JmsMessageFormatter.java]: @jdanekrh if you have some better idea of this. 
Maybe some hash of content might be better?

**Comment by jdanekrh** [comment]
Haven't noticed the PR. We already have option to print hash of the message. I think it is better than trimming. Or do you want to trim a message by default, at a certain size? Or possibly switch to printing hash starting at certain size?

**Comment by michalxo** [comment]
What exactly is hash of the message? Whole message, or just message body content? I would suggest having a hash of content would be the best (instead of "omitted" message).

**Comment by michalxo** [comment]
I see that there is already `msg-content-hashed`.  I did not know that. And actually ended with the same implementation as you have, but used SHA-256. Closing this PR. :)

**Pull request closed without merging**